scipy/stats/_hypotests.py

```diff
@@ -146,3 +146,334 @@ def _get_wilcoxon_distr(n):
                          "statistic is not implemented for n={}".format(n))
 
     return np.array(cnt, dtype=int)
+
+
+def _ct_to_lists(table):
+    """Convert contingency table to equivalent lists of rankings"""
+    table = np.array(table, copy=False, dtype=int)
+    n = table.sum()
+    x = np.zeros(n, dtype=int)
+    y = np.zeros(n, dtype=int)
+    k = 0
+    for i, row in enumerate(table):
+        for j, val in enumerate(row):
+            x[k:k+val] = i + 1
+            y[k:k+val] = j + 1
+            k += val
+    return x, y
+
+
+def _lists_to_ct(x, y):
+    """Convert lists of rankings to contingency table"""
```

Comment:
I seem to remember there being a mistake in here. In any case, I think there is existing code for creating a contingency table that I should use.